src/modules/Course/routes/index.tsx

```diff
@@ -1,38 +1,40 @@
+import React from 'react';
+import { RouteObject } from 'react-router-dom';
+
 import DefaultLayout from '../../../layouts/Default';
-import { RouteProps as Route } from '../../../router/RouteWrapper';
 import CourseClassrooms from '../containers/CourseClassrooms';
 import CourseClassroomsSubscription from '../containers/CourseClassroomsSubscription';
 import CourseMatrix from '../containers/CourseMatrix';
 import CourseOverview from '../containers/CourseOverview';
 
-const routes: Route[] = [
-  {
-    path: '/course/:course',
-    name: 'Course Overview',
-    component: CourseOverview,
-    layout: DefaultLayout,
-    exact: true,
-  },
-  {
-    path: '/course/:course/classrooms',
-    name: 'Course Classrooms',
-    component: CourseClassrooms,
-    layout: DefaultLayout,
-    exact: true,
-  },
-  {
-    path: '/course/:course/classrooms/subscription',
-    name: 'Course Classrooms Subscription',
-    component: CourseClassroomsSubscription,
-    layout: DefaultLayout,
-    exact: true,
-  },
+const routes: RouteObject[] = [
   {
-    path: '/course/:course/matrix',
-    name: 'Course Matrix',
-    component: CourseMatrix,
-    layout: DefaultLayout,
-    exact: true,
+    id: 'courses-module',
+    element: <DefaultLayout />,
```

Comment:
I'm converting the `PageLayout` component into a real `Layout` so we can use it directly here, without the need to import if at every single page.

Reply:
By the way, it will be addressed in a separate PR (to avoid HUGE conflicts).

Reply:
I strongly disagree, it's 2 different types of layout.
The external layout should be composed in the root route and be reused by all LMS containers, the ContainerLayout .i.e PageLayout, which IMO doesn't make sense be called as PageLayout since react handle much more with containers and not pages., should be used as the internal layout required by the route.